### ISSUE TITLE
Deployment on AWS fails waiting for persistent volume

### ISSUE THREAD

**Issue by aktech** [Quansight/qhub#17]
### Steps to reproduce:

- Render the aws project:
- Run `terraform init` then `terraform apply` in the `infrastructure`

First I got this:

```
Error: namespaces "dev" not found

  on .terraform/modules/kubernetes-conda-store-mount/modules/kubernetes/nfs-mount/main.tf line 38, in resource "kubernetes_persistent_volume_claim" "main":
  38: resource "kubernetes_persistent_volume_claim" "main" {
```

Then I did `terraform apply` again, then I got the following every-time:

```
Error: timeout while waiting for state to become 'Bound' (last state: 'Pending', timeout: 5m0s)

  on .terraform/modules/kubernetes-conda-store-server/modules/kubernetes/services/conda-store/main.tf line 1, in resource "kubernetes_persistent_volume_claim" "main":
   1: resource "kubernetes_persistent_volume_claim" "main" {
```

**Comment by tylerpotts** [comment]
Document solution, and how to delete the persistent volume. Will leave the solution as manual so that user data is not automatically deleted.

**Comment by filippo82** [comment]
Hi all, is there any update on this issue?

**Comment by tylerpotts** [comment]
@filippo82 There is. With the newer terraform/kubernetes update, persistent volume side can be increased without being deleted/destroyed. I've verified this as of this past week

**Comment by filippo82** [comment]
Hi @tylerpotts I believe that the issue which @aktech was experiencing in June (and which I was experiencing yesterday too) was related (somehow, I think) to the order of creation of resources by Terraform. This should be now fixed by this #129. First building the Kubernets cluster with `terraform apply -auto-approve -target=module.kubernetes -target=module.kubernetes-initialization -target=module.kubernetes-ingress` and then everything else with a general `terraform apply`.

So this is issue is now fixed for me and, I believe, it is fully taken care of by `qhub deploy` and probably can be closed.

The issue I am having is to `terraform destroy` the QHub deployment. It has been a nightmare so far :/

I've opened this issue to discuss that: #144

Best,
-Filippo
